cmd/ocitools/validate.go

```diff
@@ -337,6 +337,17 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 			} else if spec.Linux.Namespaces[index].Type == rspec.MountNamespace {
 				mountExists = true
 			}
+		} else {
+			if hostCheck {
```

Comment:
On 08/17/2016 11:42 AM, W. Trevor King wrote:
> |} else if hostCheck {| to avoid a layer of nesting?
I ever thought about this.
But, it seems they should not in the seem level.
So, I did not write like this.